### DISCUSSION TITLE
Will node red update in HA to 3.0?

### DISCUSSION THREAD

**Discussion by neil1111** [hassio-addons/addon-node-red#1393] [Ideas]
Hi. I saw on YouTube that a major update to node red was just released. It includes many great new features. Here is the video link: [What’s New in Node-Red 3.0](https://youtu.be/R9bV2MytHr0)

is the version in HA scheduled to be updated, and if so, is there a timeline planned?

Thanks, this is a fabulous and much appreciated integration!
Neil

**Comment by neil1111** [comment]
Update, I see it was just pushed as ver 13.0.  Thanks to all of the developers.

**Comment by Hypnos3** [comment]
It would be nice if in the release nodes the used Node-RED Version will be show.

**Reply by frenck** [reply]
It does..
